giskard/testing/tests/data_quality.py

```diff
@@ -0,0 +1,64 @@
+"""
+Module for data quality tests.
+"""
+from giskard.ml_worker.testing.test_result import TestResult
+from giskard.ml_worker.testing.registry.decorators import test
+from giskard.datasets.base import Dataset
+
+@test(name="Data Uniqueness Test")
+def uniqueness_test(dataset: Dataset, column: str):
+    """
+    Test for checking the uniqueness of data in a column.
+
+    Args:
+        dataset (Dataset): The dataset to test.
+        column (str): The column to check for uniqueness.
+
+    Returns:
+        TestResult: The result of the test.
+    """
+    column_data = dataset.df[column]
+    uniqueness_ratio = len(column_data.unique()) / len(column_data)
+    return TestResult(passed=uniqueness_ratio == 1)
+
+@test(name="Data Completeness Test")
+def completeness_test(dataset: Dataset):
+    """
+    Test for checking the completeness of data in a dataset.
+
+    Args:
+        dataset (Dataset): The dataset to test.
+
+    Returns:
+        dict: A dictionary with the completeness score for each column.
+    """
+    completeness_scores = {}
+    for column in dataset.df.columns:
+        column_data = dataset.df[column]
+        completeness_ratio = len(column_data.dropna()) / len(column_data)
+        completeness_scores[column] = completeness_ratio
+    return completeness_scores
```

Comment:
Giskard test must return a `TestResult`:  https://docs.giskard.ai/en/latest/open_source/customize_tests/test_model/index.html#create-execute-your-own-test

Reply:
I want to return a dictionary, which has completeness ratio for each column. Should I just add a parameter for column name and a threshold for completeness ratio. Then the test returns if it passed or not with the metric as the completeness ratio.